OfBorg eval is breaking on `beanhub-cli`:

```
FAILED tests/test_post_processor.py::test_extract_imported_transactions[simple-transactions-expected0]
```

Can you investigate? If it's a flaky test, please disable it using `disabledTests` (see [docs](https://nixos.org/manual/nixpkgs/unstable/#using-pytestcheckhook)).

--- a/pkgs/development/python-modules/beanhub-cli/default.nix
+++ b/pkgs/development/python-modules/beanhub-cli/default.nix
@@ -25,7 +25,7 @@
 
 buildPythonPackage rec {
   pname = "beanhub-cli";
-  version = "1.1.3";
+  version = "1.2.2";
   pyproject = true;
 
   disabled = pythonOlder "3.10";
@@ -34,7 +34,7 @@ buildPythonPackage rec {
     owner = "LaunchPlatform";
     repo = "beanhub-cli";
     rev = "refs/tags/${version}";
-    hash = "sha256-vYBbaUVJAs+aIp6aQpJb62DEDxe/sQTzgOkjPq6ADoc=";
+    hash = "sha256-ZN384KN6+pkDDsCk825sr+LsArQ5Vaf09ASLN8IemhM=";
   };
 
   build-system = [ poetry-core ];

--- a/pkgs/development/python-modules/beanhub-extract/default.nix
+++ b/pkgs/development/python-modules/beanhub-extract/default.nix
@@ -4,13 +4,15 @@
   buildPythonPackage,
   pythonOlder,
   pytestCheckHook,
+  iso8601,
   poetry-core,
+  pytest-lazy-fixture,
   pytz,
 }:
 
 buildPythonPackage rec {
   pname = "beanhub-extract";
-  version = "0.0.7";
+  version = "0.1.3";
   pyproject = true;
 
   disabled = pythonOlder "3.9";
@@ -19,14 +21,20 @@ buildPythonPackage rec {
     owner = "LaunchPlatform";
     repo = "beanhub-extract";
     rev = "refs/tags/${version}";
-    hash = "sha256-Wt8ZCyCaERNXEd0/Q89QWUW/wGFSHAP2RZLhnv5xkgY=";
+    hash = "sha256-Uw9bSVOpiIALkgA77OrqAcDWcEafVSnp4iILa4Jkykc=";
   };
 
   build-system = [ poetry-core ];
 
-  dependencies = [ pytz ];
+  dependencies = [
+    iso8601
+    pytz
+  ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
+  nativeCheckInputs = [
+    pytest-lazy-fixture
+    pytestCheckHook
+  ];
 
   pythonImportsCheck = [ "beanhub_extract" ];
 

--- a/pkgs/development/python-modules/beanhub-import/default.nix
+++ b/pkgs/development/python-modules/beanhub-import/default.nix
@@ -16,7 +16,7 @@
 
 buildPythonPackage rec {
   pname = "beanhub-import";
-  version = "0.1.5";
+  version = "0.3.2";
   pyproject = true;
 
   disabled = pythonOlder "3.9";
@@ -25,7 +25,7 @@ buildPythonPackage rec {
     owner = "LaunchPlatform";
     repo = "beanhub-import";
     rev = "refs/tags/${version}";
-    hash = "sha256-BH+d7cKC83b7T2Z1Xlwpsv3KACo1vDn6v65tElhir/c=";
+    hash = "sha256-jDZUsV5yj4h7zUA5NB/4exhPj3f3put/bKXZmPz94OQ=";
   };
 
   build-system = [ poetry-core ];

